website/content/en/preview/AWS/provisioning.md

```diff
@@ -252,6 +252,46 @@ spec:
 You can control the UserData that needs to be applied to your worker nodes via this field. Review the [Custom UserData documentation](../user-data/) to learn the necessary steps
 If you need to specify a launch template in addition to UserData, then review the [Launch Template documentation](../launch-templates/) instead and utilize the `spec.providerRef.launchTemplate` field.
 
+### AMISelector
+
+AMISelector is used to configure custom AMIs for Karpenter to use, where the AMIs are discovered through [AWS tags](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html), similar to `subnetSelector`. This field is optional, and Karpenter will use the latest EKS-optimized AMIs if an amiSelector is not specified.
+
+EC2 AMIs may be specified by any AWS tag, including `Name`. Selecting tag values using wildcards (`*`) is supported.
+
+EC2 AMI IDs may be specified by using the key `aws-ids` and then passing the IDs as a comma-separated string value.
+
+* When launching nodes, Karpenter automatically determines which architecture a custom AMI is compatible with and will use images that match an instanceType's requirements.
+* If multiple AMIs are found that can be used, Karpenter will randomly choose any one.

```

Comment:
Should we choose latest instead of randomly? Or oldest, perhaps?

Reply:
This is interesting. I think latest might be better than randomly but it can be risky regardless since your latest AMI may be visible, but not ready for use etc.

If I were setting this up, my tags would probably be a pointer to a specific version so I've got tight control over the exact AMI being used in all my environments. Something like - 
```
eks.aws/ami-version: v1.21.5-20220708
```

so that way I can re-use the same amiSelector in all regions *and* I've limited my provisioners to always create nodes of my approved and tested version. 

I'm happy to change the logic to `latest` by default if you think that's better.